src/main/java/org/mockito/BDDMockito.java

```diff
@@ -346,10 +346,18 @@ public void shouldHaveNoMoreInteractions() {
         /**
          * See original {@link Stubber#doNothing()}
          * @since 1.8.0
+         * @deprecated please use {@link #willDoNothing()} instead
          */
+        @Deprecated
         BDDStubber willNothing();
 
         /**
+         * See original {@link Stubber#doNothing()}
+         * @since 1.10.20
```

Comment:
Please notice this line.
I think this change should go in 1.x releases.
Or please let me know what version I should enter.